css/mobile_first.css

```diff
@@ -241,10 +245,8 @@ h3 {
 }
 
 #about-me-section-4 {
-  margin-bottom: 30px;
   border: none;
   width: 314px;
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you make use of percentages for widths instead of px as this is ensure your screen with adjusts effectively for other device. Kindly see a video below for more explanation.

https://www.loom.com/share/d11115e0d85d48f6ba73427940dfd5b7